src/Symfony/Component/AssetMapper/Command/ImportMapOutdatedCommand.php

```diff
@@ -76,6 +76,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $packagesUpdateInfos = $this->updateChecker->getAvailableUpdates($packages);
         $packagesUpdateInfos = array_filter($packagesUpdateInfos, fn ($packageUpdateInfo) => $packageUpdateInfo->hasUpdate());
         if (0 === \count($packagesUpdateInfos)) {
+            $io->writeln('Everything up to date');
```

Comment:
```suggestion
            $io->writeln('Everything is up to date');
```

This sounds more natural to me.